exchanges/coinbaseinternational/coinbaseinternational_wrapper.go

```diff
@@ -0,0 +1,805 @@
+			CryptoChain:  history.Results[j].NetworkName,
+			Timestamp:    history.Results[j].CreatedAt,
+			Status:       history.Results[j].Status,
+			Currency:     history.Results[j].Asset,
+			Amount:       history.Results[j].Amount,
+			TransferType: history.Results[j].Type,
+		}
+	}
+	return resp, nil
+}
+
+// GetWithdrawalsHistory returns previous withdrawals data
+func (co *CoinbaseInternational) GetWithdrawalsHistory(ctx context.Context, _ currency.Code, a asset.Item) ([]exchange.WithdrawalHistory, error) {
+	if a != asset.Spot {
+		return nil, asset.ErrNotSupported
+	}
+	history, err := co.ListMatchingTransfers(ctx, "", "", "", "WITHDRAW", 0, 0, time.Time{}, time.Time{})
+	if err != nil {
+		return nil, err
+	}
+	resp := make([]exchange.WithdrawalHistory, len(history.Results))
+	for j := range history.Results {
+		resp[j] = exchange.WithdrawalHistory{
+			Status:       history.Results[j].Status,
+			Timestamp:    history.Results[j].CreatedAt,
+			Currency:     history.Results[j].Asset,
+			Amount:       history.Results[j].Amount,
+			TransferType: history.Results[j].Type,
+			CryptoTxID:   history.Results[j].TransferUUID,
+			CryptoChain:  history.Results[j].NetworkName,
+		}
+	}
+	return resp, nil
+}
+
+// GetRecentTrades returns the most recent trades for a currency and asset
+func (co *CoinbaseInternational) GetRecentTrades(_ context.Context, _ currency.Pair, _ asset.Item) ([]trade.Data, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// GetHistoricTrades returns historic trade data within the timeframe provided
+func (co *CoinbaseInternational) GetHistoricTrades(context.Context, currency.Pair, asset.Item, time.Time, time.Time) ([]trade.Data, error) {
+	return nil, common.ErrNotYetImplemented
+}
+
+// GetServerTime returns the current exchange server time.
+func (co *CoinbaseInternational) GetServerTime(_ context.Context, _ asset.Item) (time.Time, error) {
+	return time.Time{}, common.ErrNotYetImplemented
+}
+
+// SubmitOrder submits a new order
+func (co *CoinbaseInternational) SubmitOrder(ctx context.Context, s *order.Submit) (*order.SubmitResponse, error) {
+	if err := s.Validate(); err != nil {
+		return nil, err
+	}
+	oType, err := orderTypeString(s.Type)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: possible value GTT is not represented by the order.Submit
+	// therefore we could not represent 'GTT'
+	var tif string
+	if s.ImmediateOrCancel {
+		tif = "IOC"
+	} else {
+		tif = "GTC"
+	}
+	response, err := co.CreateOrder(ctx, &OrderRequestParams{
+		ClientOrderID: s.ClientOrderID,
+		Side:          s.Side.String(),
+		BaseSize:      s.Amount,
+		Instrument:    s.Pair.String(),
+		OrderType:     oType,
+		Price:         s.Price,
+		StopPrice:     s.TriggerPrice,
+		PostOnly:      s.PostOnly,
+		TimeInForce:   tif,
+	})
+	if err != nil {
+		return nil, err
+	}
+	oStatus, err := order.StringToOrderStatus(response.OrderStatus)
+	if err != nil {
+		return nil, err
+	}
+	return &order.SubmitResponse{
```

Comment:
Suggestion: can use method on S to get default order.SubmitResponse if you want.